### DISCUSSION TITLE
2.4.20 won't load after reboot System appears to be starting. No highstate has completed since the system was restarted

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#11481] [2.4] [answered]
### Version

2.4.20

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

installation

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

64gig

### Storage for /

500gig

### Storage for /nsm

8tb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Patched to new version 2.4.20 all logging stopped for Palo firewall 
I tried to fix the Palo index template as it was showing so-log
it would not take the known. fix that I had applied in the past.
So I decided to reboot 
SO will not come back up 
only indicator I have is that the highstate does't get logged properly our something as I can run salt-call state.highstate, it completes elastic search connected and that. is nothing comes up

logs for last high state remain empty

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by Phillip-Samson** [comment, marked as answer]
Removed the version reference for 2.4.20 replaced with 2.4.10 from SLS files and deleted temp install files
re-ran soup.
system is back up

**Comment by defensivedepth** [comment]
@Bal33p For clarity, what custom config was in place prior to running soup?  And did soup complete without errors?
